mmv1/third_party/terraform/services/integrationconnectors/resource_integration_connectors_managed_zone_test.go

```diff
@@ -0,0 +1,79 @@
+package integrationconnectors_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+)
+
+func TestAccIntegrationConnectorsManagedZone_integrationConnectorsManagedZoneExample_update(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckIntegrationConnectorsManagedZoneDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccIntegrationConnectorsManagedZone_integrationConnectorsManagedZoneExample_full(context),
+			},
+			{
+				ResourceName:            "google_integration_connectors_managed_zone.samplemanagedzone",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"name", "labels", "terraform_labels"},
+			},
+			{
+				Config: testAccIntegrationConnectorsManagedZone_integrationConnectorsManagedZoneExample_update(context),
+			},
+			{
+				ResourceName:            "google_integration_connectors_managed_zone.samplemanagedzone",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"name", "labels", "terraform_labels"},
+			},
+		},
+	})
+}
+
+func testAccIntegrationConnectorsManagedZone_integrationConnectorsManagedZoneExample_full(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+data "google_project" "test_project" {
+}
+
+resource "google_integration_connectors_managed_zone" "samplemanagedzone" {
+  name     = "tf-test-test-managed-zone%{random_suffix}"
+  description = "tf created description"
+  labels = {
+    intent = "example"
+  }
+  target_project="connectors-example"
```

Comment:
This is to create a project in this same terraform config?

Reply:
We need a list of projects in which this may run, so that I can create IAM policies to allow the role in the project which has this resource. It didn't make sense to create end to end of this setup in terraform as this goes to terraform docs as well, please suggest.

Reply:
Hi @rileykarson please take a look

Reply:
> It didn't make sense to create end to end of this setup in terraform as this goes to terraform docs as well, please suggest.

That should be fine, we do that level of setup in those samples in other products too.

Reply:
I'd recommend the full setup (including the target project) inline.

Reply:
Not just the target project, do we allow creating a new project, if so, which billing account to use, etc,etc

Reply:
From chat:

> You draw them from the environment! https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/services/apigee/resource_apigee_environment_type_test.go.erb#L13 is a good reference test